package.json

```diff
@@ -134,10 +139,28 @@
     "onCommand:service.explorer.reportIssue",
     "onCommand:service.explorer.apply",
     "onCommand:service.explorer.deleteLocal",
-    "onCommand:service.output"
+    "onCommand:service.output",
+    "onCommand:function.delete",
+    "onCommand:function.explorer.refresh",
+    "onCommand:function.explorer.create",
+    "onCommand:function.build.Palette",
```

Comment:
in the command palette, I want to show `function build/deploy` it will look something
<img width="730" alt="Screenshot 2021-12-21 at 2 38 16 PM" src="https://user-images.githubusercontent.com/9924513/146902867-9d0248ee-3c05-423f-ba60-780fff327816.png">
 like